File: samples/java/src/main/java/com/function/RedisStreamTrigger/FastPollingStreamTrigger.java
package com.function.RedisStreamTrigger;

import com.microsoft.azure.functions.*;
import com.microsoft.azure.functions.annotation.*;
import com.microsoft.azure.functions.redis.annotation.*;

public class FastPollingStreamTrigger {
    @FunctionName("FastPollingStreamTrigger")
    public void run(
            @RedisStreamTrigger(
                name = "entry",
                connectionStringSetting = "redisLocalhost",
                key = "streamTest",
                pollingIntervalInMs = 100,
                maxBatchSize = 1)
                String message,
            final ExecutionContext context) {
            context.getLogger().info(message);
    }
}

File: samples/dotnet-isolated/RedisStreamTrigger/FastPollingStreamTrigger.cs
using Microsoft.Extensions.Logging;

namespace Microsoft.Azure.Functions.Worker.Extensions.Redis.Samples.RedisStreamTrigger
{
    internal class FastPollingStreamTrigger
    {
        private readonly ILogger<FastPollingStreamTrigger> logger;

        public FastPollingStreamTrigger(ILogger<FastPollingStreamTrigger> logger)
        {
            this.logger = logger;
        }

        [Function(nameof(FastPollingStreamTrigger))]
        public void Run(
            [RedisStreamTrigger(Common.connectionStringSetting, "streamKey", pollingIntervalInMs: 100)] string entry)
        {
            logger.LogInformation(entry);
        }
    }
}
